examples/configurations/private-link-schema-registry/main.tf

```
@@ -248,7 +248,7 @@ resource "confluent_schema" "order" {
   schema_registry_cluster {
     id = data.confluent_schema_registry_cluster.essentials.id
   }
-  rest_endpoint = data.confluent_schema_registry_cluster.essentials.private_rest_endpoint
+  rest_endpoint = data.confluent_schema_registry_cluster.essentials.private_region_rest_endpoints[local.region]
```

```suggestion
  rest_endpoint = data.confluent_schema_registry_cluster.essentials.private_regional_rest_endpoints[local.region]
```

and let's try to run it, I'm a bit afraid this `rest_endpoint` could be evaluated to `""`.

---

we would need the help of SR team in running this i guess as they have the private environment setups